Gems/AtomLyIntegration/CommonFeatures/Code/Source/SharedPreview/SharedThumbnail.cpp

```diff
@@ -44,6 +43,7 @@ namespace AZ
         void SharedThumbnail::LoadThread()
         {
             m_state = State::Loading;
+            AzToolsFramework::Thumbnailer::ThumbnailerRendererNotificationBus::Handler::BusConnect(m_key);
```

Comment:
Is this bus mutex-protected? I don't think it is, but it needs to be if you are connecting on another thread.